selfdrive/car/toyota/carcontroller.py

```diff
@@ -22,6 +26,8 @@ def __init__(self, dbc_name, CP, VM):
     self.standstill_req = False
     self.steer_rate_limited = False
 
+    self.steer_rate_counter = 0
+
     self.packer = CANPacker(dbc_name)
```

Comment:
```suggestion
    self.standstill_req = False
    self.steer_rate_limited = False
    self.steer_rate_counter = 0

    self.packer = CANPacker(dbc_name)
```
whitespace nitpick maybe?